app/components/message_attachments/action_button/action_button.js

```diff
@@ -49,13 +52,17 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
         button: {
             borderRadius: 2,
             backgroundColor: theme.buttonBg,
+            opacity: 1,
             alignItems: 'center',
             marginBottom: 2,
             marginRight: 5,
             marginTop: 10,
             paddingHorizontal: 10,
             paddingVertical: 7,
         },
+        buttonDisabled: {
+            backgroundColor: theme.buttonBg + '4F',
```

Comment:
@migbot This does not work. The RN button component uses opacity to make the button "flash" when clicked. If I use the opacity here to change it to look disabled, for some reason it will still "flash" when clicked.

Reply:
Oh wait. NVM I just realized what you meant. Will change.

Reply:
done.